app/models/user.rb

```diff
@@ -61,6 +61,12 @@ class User < ApplicationRecord
   }
   scope :active_users, -> { where(deleted_at: nil) }
 
+  ROLE_WEIGHTINGS = {
```

Comment:
Let's discuss how this could/should work

Reply:
Well spotted - didn't even use the weightings for the `extend_sar_deadline` work! Will amend!

Reply:
I've left the additional changes to `User.rb` in place and additionally parameterised `role_weightings` in User#self.sort_teams_by_roles`.

This is to prevent code duplication for `User#team_for_case` used in  `CaseExtendSARDeadlineService`, `CaseRemoveSARDeadlineExtensionService` & `MessagesController` at which point it makes sense to move the weighting/role sorting code in to the `User` model as well for consistency.